gcc/rust/ast/rust-expr.h

```diff
@@ -4731,13 +4863,44 @@ struct TupleTemplateStr
 // Inline Assembly Node
 class InlineAsm : public ExprWithoutBlock
 {
+private:
+  location_t locus;
+  // TODO: Not sure how outer_attrs plays with InlineAsm, I put it here in order
+  // to override, very hacky.
+  std::vector<Attribute> outer_attrs;
```

Comment:
what exactly was the issue here? parsing?

Reply:
i'm just not sure what outer_attrs mean in general and how it plays out with asm!

Reply:
ah, alright. can you open an issue for this please? titled something like `Handle outer attributes properly for inline assembly` and we'll have a look later during the project :)